packages/cloud-cognitive/src/components/InlineEdit/InlineEdit.test.js

```diff
@@ -75,4 +76,197 @@ describe(componentName, () => {
       componentName
     );
   });
+
+  it('can start and stop edit with focus', async () => {
+    const startingValue = value;
+    const { container } = render(
+      <InlineEdit {...requiredProps} value={startingValue} />
+    );
+
+    const input = screen.getByRole('textbox');
+    userEvent.click(input);
+    const component = container.querySelector(`.${blockClass}`);
+    // clicks in
+    expect(component).toHaveClass(`${blockClass}--editing`);
+
+    // tabs out
+    userEvent.tab({ shift: true });
+    expect(component).not.toHaveClass(`${blockClass}--editing`);
+
+    // tabs in
+    userEvent.tab();
+    expect(component).toHaveClass(`${blockClass}--editing`);
+
+    // types some text
+    userEvent.keyboard('123');
+    expect(input).toHaveTextContent(`${startingValue}123`);
+
+    // testing {ctrl}z does not seem to work
+    // userEvent.type(input, '{ctrl}z');
+    // expect(input).toHaveTextContent(startingValue);
+  });
+
+  it('can start and cancel an edit using buttons', async () => {
+    const startingValue = value;
+    const { container } = render(
+      <InlineEdit {...requiredProps} value={startingValue} />
+    );
+    const component = container.querySelector(`.${blockClass}`);
+
+    const input = screen.getByRole('textbox');
+
+    // starts edit
+    const edit = container.querySelector(`.${blockClass}__edit`);
+    userEvent.click(edit);
+    expect(component).toHaveClass(`${blockClass}--editing`);
+
+    // has edit buttons
+    let controls = screen.getAllByRole('button');
+    expect(controls).toHaveLength(2); // cancel and save
+
+    // click cancel check result reverts
+    userEvent.click(controls[0]);
+    expect(input).toHaveTextContent(startingValue);
+    expect(component).not.toHaveClass(`${blockClass}--editing`);
+  });
+
+  it('can start and save edit with buttons', async () => {
```

Comment:
You don't need the async as you're not doing any awaits.

```suggestion
  it('can start and save edit with buttons', () => {
```